### DISCUSSION TITLE
SSL证书申请接口被墙问题

### DISCUSSION THREAD

**Discussion by chivehao** [1Panel-dev/1Panel#6769] [Q&A]
### 1Panel 版本

v1.10.19-lts

### 问题描述

我看申请证书的时候是访问的这个域名：` AuthURL: https://acme-v02.api.letsencrypt.org/acme/xxx`
但是这个域名被墙了好像，我只有在路由器开代理，走代理才能正常成功
![image](https://github.com/user-attachments/assets/7fb7b01a-dc05-46f3-9b15-9ec5cb6acaff)

### 重现步骤

找台国内的机器，申请SSL证书

### 期待的正确结果

能正常成功申请证书

**Comment by zhengkunwang223** [comment]
![image](https://github.com/user-attachments/assets/3a93322d-fd91-4926-a123-58f5b8326c36)
没被墙
